### PR TITLE
Cloud 210 backup docs

### PR DESCRIPTION
**Pull request by patrickbirch**
Updated backup topic to remove punctuation.

### CODE REVIEW

File: source/backups.rst [hunk 1]
```diff
@@ -148,7 +148,7 @@ Following steps are needed to restore a previously saved backup:
 
    ::
 
-      ./deploy/backup/restore-backup.sh <backup-name> <cluster-name>
+      kubectl apply -f deploy/backup/restore.yaml
```
Comment (delgod) [inline source/backups.rst]: Could you also add an example like in lines 111-124

Comment (patrickbirch) [reply source/backups.rst]: Updated.

Comment (delgod) [reply source/backups.rst]: sorry, still don't see example 😞

**Comment by delgod** [comment]
@patrickbirch you copy-pasted example of Backup, it is Recovery documentation.
please take content from recovery.yaml

**Comment by patrickbirch** [comment]
OK.

Patrick Birch, Senior Technical Writer,
Percona
Timezone: CST/CDT (UTC-6/UTC-5)
Tel: +1 (713) 460-0201
Mobile: +1 (832) 266-5186

**Comment by patrickbirch** [comment]
Updated text to match restore.yaml in the example from lines 154-167.

**Pull request merged**